packages/devtools_app/lib/src/timeline/flutter/timeline_processor.dart

```diff
@@ -40,7 +40,11 @@ StringBuffer debugFrameTracking = StringBuffer();
 
 const String gpuEventName = 'GPURasterizer::Draw';
 
-const String uiEventName = 'VSYNC';
+// For older versions of Flutter, the starting event for the UI portion of a
+// Flutter frame was 'VSYNC'. We need to check for both so that we can still
+// support users on older versions of Flutter.
+const String uiEventNameOld = 'VSYNC';
+const String uiEventName = 'VsyncProcessCallback';
```

Comment:
Hmm, `VSYNC` seems more semantic. It'd be good to get this under test in the engine (as your bug mentions).